sample-extractors/image-annotation-extractor/image_annotation.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+"""Example extractor based on the clowder code."""
+
+import logging
+import subprocess
+import json
+from typing import Dict
+
+from pyclowder.extractors import Extractor
+import pyclowder.files
+
```

Comment:
Can this name be refactored from 'Wordcount' to the name of this extractor? Thanks.

Reply:
Thanks for the catch!